components/messaging/CurrentConversation.tsx

```diff
@@ -13,25 +13,16 @@ const CurrentConversation: React.FC = () => {
   const [currentMessages, setCurrentMessages] = useState<MessageType[]>([]);
 
   useEffect(() => {
-    const highestId = Math.max(
-      ...allConversations.map((conversation) => conversation.conversation_id)
-    );
-
-    setCurrentConversation &&
-      setCurrentConversation(
-        allConversations?.filter(
-          (conversation) => conversation.conversation_id === highestId
-        )[0]
-      );
-  }, []);
+    setCurrentConversation && setCurrentConversation(allConversations[0]);
+  }, [allConversations, setCurrentConversation]);
 
```

Comment:
I have the feeling this is maybe not correct. Why do we want these dependency? We don't want this use effect to run when these things change, as that would reset the current convo when a new conversation is added or anytime a user clicked to open a new conversation. I think we just want this run once on mount. So have no dependencies.

Reply:
Marking this as resolved following our conversation :)